resources/js/Shared/Layout/Nav.vue

```diff
@@ -48,6 +48,18 @@ function login() {
   })
 }
 
+function githubLogin() {
```

Comment:
Actually I prefer function with button over <a> tag, because implementation of toastification would be more difficult. I'll add this functionalty in next commit.